app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    source = command.split()
+    if len(source) != 3 and source[0] != "cp":
```

Comment:
The condition `len(source) != 3 and source[0] != 'cp'` should be `len(source) != 3 or source[0] != 'cp'`. The current condition will return if either the length is not 3 or the command is not 'cp', which is incorrect. It should return if either condition is true.